.github/workflows/pull-request-test.yml

```
@@ -77,6 +77,8 @@ jobs:
         RENKU_BOT_DEV_PASSWORD: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
         RENKU_ANONYMOUS_SESSIONS: true
         RENKU_TESTS_ENABLED: true
+        OS_DEV_ACCESS: ${{ secrets.OS_DEV_ACCESS }}
```

I would rename the access and secret S3 keys to start with `S3` rather than `OS`

```suggestion
        S3_DEV_ACCESS: ${{ secrets.S3_DEV_ACCESS }}
```

---

Done.